skills/gh-address-comments/scripts/mark_resolved.py

```
@@ -157,6 +163,24 @@ def main() -> None:
             parser.error("Provide at least one THREAD_ID or use --all --input FILE")
 
     failures = run_resolution(thread_ids, args.dry_run, args.workers)
+
+    # Auto-close beads for resolved threads (skip silently if bd not ready or no mapping)
+    if not args.no_beads and not failures and args.input:
+        if args.dry_run or check_bd_ready(fatal=False):
+            base, _ = os.path.splitext(args.input)
+            mpath = base + ".beads.json"
+            if os.path.exists(mpath):
+                print(f"\nClosing beads...")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)